src/Button/Button.js

```diff
@@ -1,24 +1,27 @@
 import React from 'react'
 import propTypes from 'prop-types'
+import { buttonStyles, buttonSizes } from './Button.levers'
 
 /**
- * Primary UI component for user interaction
+ * Button Component
+ *
+ * HTML button element
  */
-export const Button = ({ className, children, onClick, type, ref, ...props }) => {
+export const Button = ({ classes, children, onClick, style, size, ref, ...props }) => {
```

Comment:
I have to say, it felt more natural to write `className` on this component but it does break a bit from the convention on the alert nested object. We should have a convention for this. My rough thinking was that one level components like button, link, could use className, whereas the more complicated components could use the large object.

Reply:
Yeah, I'm ok with that